typeql/language/insert.feature

```diff
@@ -238,8 +238,6 @@ Feature: TypeQL Insert Query
       insert $x isa thing;
       """
 
-
-
```

Comment:
Did you mean to do this?
We seem to keep two 2 lines between scenarios and section headers. Was this a fix for that or does it break it?